### PR TITLE
PD-584: Update Logos in ProjectNav and UserMenu

### PR DESCRIPTION
**Pull request by brookescarlett**
## ✍️ Proposed changes

- Update logos in ProjectNav and UserMenu

🎟 _Jira ticket:_ [PD-584](https://jira.mongodb.org/browse/PD-584)

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

Screen Shot 2020-03-20 at 12 51 26 PM

Screen Shot 2020-03-20 at 4 55 06 PM

Screen Shot 2020-03-20 at 4 55 16 PM

### CODE REVIEW

File: packages/mongo-nav/src/project-nav/ProjectNav.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ import {
   ActiveNavElement,
   NavElement,
 } from '../types';
+import { Atlas, RealmActive, RealmInactive, Charts } from '../helpers/Icons';
```
Comment (hswolff) [inline packages/mongo-nav/src/project-nav/ProjectNav.tsx]: Wonder if we should suffix all these names with `Icon` to clarify their usage.

File: packages/mongo-nav/src/project-nav/ProjectNav.tsx [hunk 2]
```diff
@@ -344,10 +345,10 @@ export default function ProjectNav({
             }`}
           >
             {!isMobile && (
-              <Icon
+              <Atlas
+                active={activeProduct === Product.Cloud && !!current}
```
Comment (hswolff) [inline packages/mongo-nav/src/project-nav/ProjectNav.tsx]: Do we need the double `!!` or not? Seems to be inconsistently applied.

File: packages/mongo-nav/src/user-menu/UserMenu.tsx [hunk 3]
```diff
@@ -251,11 +252,11 @@ function UserMenu({
     ? `${account.firstName ?? ''} ${account.lastName ?? ''}`
     : '';
 
-  const isAccount = activeProduct === 'account';
-  const cloudProducts = ['cloud', 'stitch', 'charts'];
+  const isAccount = activeProduct === Product.Account;
+  const cloudProducts = ['cloud', 'realm', 'charts'];
```
Comment (hswolff) [inline packages/mongo-nav/src/user-menu/UserMenu.tsx]: These can also use the enum values?

File: packages/mongo-nav/src/project-nav/ProjectNav.tsx (hunk 1 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/project-nav/ProjectNav.tsx]: 👍

File: packages/mongo-nav/src/project-nav/ProjectNav.tsx (hunk 2 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/project-nav/ProjectNav.tsx]: We definitely do for the project nav IconButtons where it wasn't previously applied, but actually not sure if we need to disable these tabs in the loading state. I'll talk to Nelle and see

File: packages/mongo-nav/src/user-menu/UserMenu.tsx (hunk 3 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/user-menu/UserMenu.tsx]: Typecasted, nvm 👍

File: packages/mongo-nav/src/project-nav/ProjectNav.tsx (hunk 2 above)
Comment (brookescarlett) [reply packages/mongo-nav/src/project-nav/ProjectNav.tsx]: Confirmed this logic is correctly applied everywhere :D

File: packages/mongo-nav/src/project-nav/ProjectNav.tsx [hunk 4]
```diff
@@ -430,7 +435,9 @@ export default function ProjectNav({
                 className={iconButtonMargin}
                 size="large"
                 disabled={!current}
-                active={activeNav === ActiveNavElement.ProjectNavInvite}
+                active={
+                  activeNav === ActiveNavElement.ProjectNavInvite && !!current
```
Comment (hswolff) [inline packages/mongo-nav/src/project-nav/ProjectNav.tsx]: May want to cache the `!!current` variable.

Comment (brookescarlett) [reply packages/mongo-nav/src/project-nav/ProjectNav.tsx]: Done 👍

**Pull request merged**